src/ledger/LedgerManagerImpl.h

```diff
@@ -197,6 +197,7 @@ class LedgerManagerImpl : public LedgerManager
 
 #ifdef BUILD_TESTS
     std::vector<TransactionMetaFrame> mLastLedgerTxMeta;
+    std::optional<LedgerCloseMetaFrame> mLastLedgerCloseMeta;
```

Comment:
We should be able to remove `mLastLedgerTxMeta`, but I didn't want to make this PR even bigger.